RecoPixelVertexing/PixelTrackFitting/plugins/PixelTrackProducer.cc

```diff
@@ -92,6 +93,14 @@ void PixelTrackProducer::store(edm::Event& ev, const TracksWithTTRHs& tracksWith
     unsigned int nHits = tracks->at(k).numberOfValidHits();
     theTrackExtra.setHits(hitCollProd, cc, nHits);
     cc +=nHits;
+    reco::TrackExtra::TrajParams trajParams;
```

Comment:
Hi @JanFSchulte - please simplify this to 
reco::TrackExtra::TrajParams trajParams(nHits,LocalTrajectoryParameters(v,1.));
reco::TrackExtra::Chi2sFive chi2s(nHits,0);
theTrackExtra.setTrajParams(std::move(trajParams),std::move(chi2s));